ppr-ui/src/views/newMhrRegistration/YourHome.vue

```diff
@@ -114,6 +138,19 @@ export default defineComponent({
       }),
       validateOther: computed(() => {
         return getSectionValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.OTHER_VALID)
+      }),
+      showHelp: computed(() => isMhrManufacturerRegistration.value),
+      showRebuiltStatus: computed(() => !isMhrManufacturerRegistration.value),
+      showOtherInformation: computed(() => !isMhrManufacturerRegistration.value),
+      manufacturerMakeModelPrompt: computed(() : string => {
+        return isMhrManufacturerRegistration.value
```

Comment:
It looks like the prompt text is based on role (manufacturer vs staff) but here, it is based on the registration flag?

Reply:
Alright I changed it to be based on the role.

Reply:
Thanks, if what I said makes sense, as I don't know the full requirements.